types/hapi-auth-basic/tsconfig.json

```diff
@@ -17,6 +17,9 @@
             "boom": [
                 "boom/v4"
             ],
+            "catbox": [
+                "catbox/v7"
```

Comment:
Need to look into updating those packages in the future as well 😄

Reply:
Well, there's the PR pending for hapi-auth-basic and nes :) I'll rebase it after this one.